$NetBSD: patch-po_Makefile.in.in,v 1.1 2017/08/16 17:54:09 adam Exp $

Bring back DESTDIR support which is somehow patched out by pkgsrc.

--- po/Makefile.in.in.orig	2011-02-27 00:39:47.000000000 +0100
+++ po/Makefile.in.in	2011-02-27 01:11:12.000000000 +0100
@@ -128,8 +128,8 @@
 	for cat in $$catalogs; do \
 	  cat=`basename $$cat`; \
 	  case "$$cat" in \
-	    *.gmo) destdir=$(gnulocaledir);; \
-	    *)     destdir=$(localedir);; \
+	    *.gmo) destdir=$(DESTDIR)$(gnulocaledir);; \
+	    *)     destdir=$(DESTDIR)$(localedir);; \
 	  esac; \
 	  lang=`echo $$cat | sed 's/\$(CATOBJEXT)$$//'`; \
 	  dir=$$destdir/$$lang/LC_MESSAGES; \
